wp-content/themes/bh/views/event/event-element.php

```diff
@@ -12,34 +12,34 @@
 $event_element =
 	"<div class='row'>" .
 		// first column - event name and date
-		"<div class='col-md-3 visible-md visible-lg col-1'>" .
+		"<div class='col-1 col-xs-12 visible-xs'>" .
 			"<h2 class='event-name'><a href='" . get_permalink($event->ID) . "'>" . get_the_title($event->ID) . "</a></h2>" .
 			$event_date .
 		"</div>" .
 		
 		// second column - event image
-		"<div class='col-xs-6 col-sm-4 col-md-3 col-2'>" .
+		"<div class='col-2 col-xs-6 col-sm-4 col-md-3'>" .
 			( ($image) ? "<a href='" . get_permalink($event->ID) . "'><img src='" . $image['sizes']['thumbnail'] . "' alt='" . ( ($image['alt']) ? $image['alt'] : '' ) . "' /></a>" : "" ) .
 		"</div>" .
 		
 		// third column - event description and buttons
-		"<div class='col-xs-6 col-sm-8 col-md-6 col-3'>" .
+		"<div class='col-3 col-xs-6 col-sm-8 col-md-9'>" .
 			"<div class='event-excerpt'>" .
 				// event name and date only for small screen size
-				"<div class='visible-xs visible-sm'>" .
+				"<div class='event-name-wrap'>" .
 					"<h2 class='event-name'><a href='" . get_permalink($event->ID) . "'>" . get_the_title($event->ID) . "</a></h2>" .
-					$event_date .
+					// "<div class='event-meta'>" . BH_get_event_type($event->ID) . $event_date . "</div>" .
 				"</div>" .
 				
 				// event description
-				"<div class='event-desc'>" . $description . "</div>" .
+				"<div class='event-desc'>" . "<div class='event-meta'>" . BH_get_event_type($event->ID) . $event_date . "</div>" .$description . "</div>" .

```

Comment:
@nirgoldberg If I remember correctly, this was because of the way this file is used: it's whole purpose is declaring `$event_element` as a variable that's used in `set-event-elements.php` where you're including the whole `event-element.php` file. Since declaring vars in this file wouldn't have helped, and since I didn't want to change the methodology of the file calling it, I used the functions.

However, is there any reason not to refactor `event-element.php` as a **function**, into which I could pass the relevant data as parameters?